### PR TITLE
[IMP] estate: add tag, type and offers T#69469

### PR DESCRIPTION
**Pull request by Suavesito-Olimpiada**
In this PR I implement offers, tags and types of property. The details are included in the commit message.

This is ready for review @deivislaya

Regards!

### CODE REVIEW

**Comment by Suavesito-Olimpiada** [comment]
All the comments have been addressed, waiting for review.

File: estate/models/estate_property.py [hunk 1]
```diff
@@ -31,29 +32,65 @@ class Property(models.Model):
...
+            ("east", "East"),
+            ("west", "West"),
+        ],
+        help="Orientation of the garden.",
+    )
+    property_type_id = fields.Many2one(
+        "estate.property.type",
+        help="Type of the property.",
+    )
+    buyer_id = fields.Many2one(
```
Comment (deivislaya) [inline estate/models/estate_property.py:74]: Add attribute copy=False

File: estate/views/estate_property_views.xml [hunk 2]
```diff
@@ -71,6 +83,7 @@
                 <field name="expected_price"/>
                 <field name="selling_price"/>
                 <field name="date_availability"/>
+                <field name="property_type_id"/>
```
Comment (deivislaya) [inline estate/views/estate_property_views.xml:86]:
```suggestion
                <field name="property_type_id"/>
                <field name="tag_ids" widget="many2many_tags" />
```

File: estate/models/estate_property.py (hunk 1 above)
Comment (Suavesito-Olimpiada) [reply estate/models/estate_property.py:74]: Done. :)

**Comment by Suavesito-Olimpiada** [comment]
Ready for review, ping @deivislaya.

File: estate/models/estate_property.py (hunk 1 above)
Comment (deivislaya) [reply estate/models/estate_property.py:74]: Pending change

File: estate/models/estate_property.py [hunk 3]
```diff
@@ -31,29 +32,66 @@ class Property(models.Model):
...
+            ("north", "North"),
+            ("south", "South"),
+            ("east", "East"),
+            ("west", "West"),
+        ],
+        help="Orientation of the garden.",
+    )
+    property_type_id = fields.Many2one(
+        "estate.property.type",
+        copy=False,
```
Comment (deivislaya) [inline estate/models/estate_property.py]: It is necessary?

Comment (Suavesito-Olimpiada) [reply estate/models/estate_property.py]: I got confused and added to the wrong field. 😅 😆

**Comment by Suavesito-Olimpiada** [comment]
I corrected my mistake and force pushed the fix, ready for review. @deivislaya

**Pull request merged**